arches/app/models/resource.py

```diff
@@ -648,11 +648,13 @@ def delete(self, user={}, index=True, transaction_id=None):
                     else False
                 )
                 if resource_is_provisional is True:
-                    creator_id = int(
-                        EditLog.objects.filter(
-                            resourceinstanceid=self.pk, edittype="create"
-                        ).values_list("userid")[0][0]
-                    )
+                    creator_id = EditLog.objects.filter(
+                        resourceinstanceid=self.pk, edittype="create"
+                    ).values_list("userid")[0][0]
+                    try:
+                        creator_id = int(creator_id)
+                    except ValueError:
+                        creator_id = ""
```

Comment:
This could just `pass` so that if this block grows later we don't wonder why the `""` was used.